lib/cookie.js

```diff
@@ -620,6 +636,43 @@ Cookie.prototype.inspect = function inspect() {
     '"';
 };
 
+Cookie.prototype.toJSON = function() {
+  var obj = {};
+
+  var props = this.serializableProperties || Cookie.serializableProperties;
```

Comment:
Hmm, will deserialization add back any of the additional fields? It looks like deserialization is using `fromJSON` and that only sets values for keys is `Cookie.serializableProperties` - I think we may lose additional fields on deserialization if I'm not mistaken.